src/main/kotlin/org/partiql/scribe/targets/trino/TrinoRewriter.kt

```diff
@@ -531,7 +537,12 @@ public open class TrinoRewriter(val onProblem: ProblemCallback) : PlanRewriter<R
             is ListType -> {
                 val head = "ARRAY<"
                 val elementType = elementType.toTrinoString()
-                head + elementType + ">"
+                "$head$elementType>"
+            }
+            is BagType -> { // map PartiQL bags to arrays
+                val head = "ARRAY<"
+                val elementType = elementType.toTrinoString()
+                "$head$elementType>"
             }
```

Comment:
Why not error? Trino does not support PartiQL bags.

Reply:
Yeah, should be an error -- was trying to get some local tests passing. Changed to give an error in [`9ae4e15` (#64)](https://github.com/partiql/partiql-scribe/pull/64/commits/9ae4e157f015c82f8882e78220a942690a47c231).